creator-node/src/services/stateMachineManager/CNodeHealthManager.ts

```diff
@@ -248,17 +248,17 @@ async function resetEarliestUnhealthyTimestamp(
 }
 
 /**
- * Returns /health_check/verbose response.
+ * Returns /health_check response.
  * TODO: - consider moving this pure function to libs
  *
  * @param {string} endpoint the endpoint to query
- * @returns {Object} the /health_check/verbose response from the endpoint
+ * @returns {Object} the response from /health_check endpoint
  */
 async function _queryVerboseHealthCheck(endpoint: string) {
```

Comment:
should we also change this function name to be _queryHealthCheck?

Reply:
yep done